ts/store/actions/wallet/payment.ts

```diff
@@ -124,6 +124,23 @@ export const paymentFetchPspsForPaymentId = createAsyncAction(
   "PAYMENT_FETCH_PSPS_FOR_PAYMENT_ID_FAILURE"
 )<PaymentFetchPspsForPaymentIdRequestPayload, ReadonlyArray<Psp>, Error>();
 
+type PaymentFetchAllPspsForPaymentIdRequestPayload = Readonly<{
+  idPayment: string;
+  idWallet: string;
+  onSuccess?: (
+    action: ActionType<typeof paymentFetchAllPspsForPaymentId["success"]>
+  ) => void;
+  onFailure?: (
+    action: ActionType<typeof paymentFetchAllPspsForPaymentId["failure"]>
+  ) => void;
```

Comment:
If not required for backward compatibility, can I suggest to remove this part? 
I think that the use of callbacks in the payload of an action can lead to the breaking of uni-direction redux / flux pattern.

Reply:
Since it is very similar to the action `paymentFetchPspsForPaymentId` it is wild copy/paste
removed in https://github.com/pagopa/io-app/pull/2091/commits/aed966f73aeec174dfe0a7717482297d15c74add